ui/language_utils.js

```diff
@@ -95,6 +98,9 @@ shaka.ui.LanguageUtils = class {
 
       span.textContent =
           shaka.ui.LanguageUtils.getLanguageName(language, localization);
+      if (forced) {
+        span.textContent += ' (' + forcedString + ')';
+      }
```

Comment:
This is mostly overwritten below and the LANGUAGE_ROLE case gets this twice.  So I think it would be better to put this after the switch so all cases get it exactly once.

Reply:
For the LABEL format it does not apply, but I have changed it to fit what you say.